docs/howto/operator/gcp/gcs.rst

```diff
@@ -1,20 +1,22 @@
-..  Licensed to the Apache Software Foundation (ASF) under one
+ .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why not make the default be 
```suggestion
..  Licensed to the Apache Software Foundation (ASF) under one
```

then we don't have to change most of the files?

Reply:
The pre-commit plugin always adds space for non-empty license lines. It works well for all other cases but for .rst files it does not. I figured that it is better to change .rst files to adapt rather than customise the plugin. The benefit of the pre-commit is that it adds licences automatically and you do not have to even think about licenses any more once you start using it, so I think it is a small price to pay with once-off space offset for .rst files. I double-checked that it works fine when documentatio.n is generated.